src/common/regions/utils.ts

```diff
@@ -6,13 +6,65 @@ import values from 'lodash/values';
 import isNull from 'lodash/isNull';
 import without from 'lodash/without';
 import regions from './region_db';
-import { getStateFips } from './regions_data';
-import { County, State, Region, MetroArea } from './types';
+import { County, State, Region, RegionType, MetroArea } from './types';
 import { GeolocationInfo } from 'common/hooks/useGeolocation';
 import { CountyToZipMap } from 'common/data';
 
 const UNITED_STATES = 'United States';
 
+// getStateName, getStateCode, and getStateFips are helper functions that make migrating
+// some of the existing state based logic over.  Ideally we will be able
+// to remove these at some point, but they are helpful in the meantime.
+
+export function getStateName(region: Region): string | null {
+  if (region.regionType === RegionType.COUNTY) {
+    return (region as County).state.fullName;
+  }
+  if (region.regionType === RegionType.STATE) {
+    return (region as State).fullName;
+  }
```

Comment:
FYI- I guess this is existing code, so feel free to leave it as-is, but you could clean it up (take advantage of TypeScript type narrowing) with:
```
if (region instanceof County) {
  return region.state.fullName;
}
if (region instanceof State) {
  return region.fullName;
}
```

and similar below...

Reply:
I'll leave it for now.